modules/salt/pages/config-modules.adoc

```diff
@@ -0,0 +1,39 @@
+[[config-modules]]
+= Configuration Modules
+
+[IMPORTANT]
+====
+This feature is a technology preview.
+====
+
+You can use Salt execution and state modules to configure your {productname} Server.
+These configuration modules can be used directly or using SLS files.
+They are are especially useful if you have multiple {productname} Servers, for example in Hub installations, but they are also useful for smaller installations.
+
+For more information about using Hub, see xref:large-deployments:multi-server.adoc[].
+
+You can use configuration modules to configure:
+
+* Organizations
+* Users
+* User permissions
+* System groups
+
+More configuration modules are planned to be released in the future.
```

Comment:
Technically speaking there is no plan set in stone yet, although I can see them coming. Not even sure if it's worth it to mention here...

Reply:
Removed.